pkg/archlinux/create-pkgbuild.sh

```diff
@@ -3,38 +3,49 @@
 VERSION=$1
 
 if [ ! "${VERSION?}" ]; then
-   echo "Usage: create-pkgbuild.sh {version}"
+   echo 'Usage: create-pkgbuild.sh {version}'
    exit 1
 fi
 
 cat <<- EOF
 pkgname=replay-sorcery
 pkgver=$VERSION
 pkgrel=1
-pkgdesc="An open-source, instant-replay screen recorder for Linux"
-arch=("i686" "x86_64")
-license=("GPL3")
-depends=("gcc-libs" "libxext")
-makedepends=("cmake" "make" "git" "nasm")
-url="https://github.com/matanui159/ReplaySorcery"
-source=("\${pkgname}::git+\${url}.git#tag=\${pkgver}")
-sha256sums=("SKIP")
+pkgdesc='An open-source, instant-replay screen recorder for Linux'
+arch=(i686 x86_64)
+license=(GPL3)
+depends=(gcc-libs libxext)
+makedepends=(cmake git nasm)
+url='https://github.com/matanui159/ReplaySorcery'
+source=("\${pkgname}"::git+"\${url}".git#tag="\${pkgver}"
+        git+https://github.com/libjpeg-turbo/libjpeg-turbo.git
+        git+https://github.com/ianlancetaylor/libbacktrace.git
+        git+https://code.videolan.org/videolan/x264.git
+        git+https://github.com/lieff/minimp4.git)
```

Comment:
Does this still respect the commit that each submodule is at?

Reply:
Yep, makepkg handles that in prepare() as can be seen here: https://gist.github.com/0a3351c32773e1e4be6cb8fa55a5d566